heliopy/util/config.py

```diff
@@ -22,14 +23,14 @@ def get_config_file():
     config_filename = 'heliopyrc'
```

Comment:
In the docstring above here, `string` needs to be changed to `Path`

Reply:
Oh yep, on it!